src/main/java/com/pubnub/api/PNConfiguration.java

```diff
@@ -201,7 +202,9 @@ public void setUserId(@NotNull UserId userId) {
     private PNReconnectionPolicy reconnectionPolicy;
 
     /**
-     * Set how many times the reconneciton manager will try to connect before giving app
+     * Set how many times the reconnection manager will try to connect before giving up.
+     * Max allowed value is 10.
+     * Default is 10.
```

Comment:
update javadoc with the change back to infinite

Reply:
Good catch.